templates/2_Pixie_Cluster_Pixels.ipynb

```diff
@@ -38,77 +38,83 @@
   },
```

Comment:
<p>Okay last thing I promise, can we change this to "Pixie: pixel clustering notebook" and the cell clustering one to "Pixie: cell clustering notebook"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/856/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>